include/oneapi/dpl/pstl/glue_memory_impl.h

```diff
@@ -47,9 +48,7 @@ uninitialized_copy(_ExecutionPolicy&& __exec, _InputIterator __first, _InputIter
 
     auto __dispatch_tag = oneapi::dpl::__internal::__select_backend(__exec, __first, __result);
 
-    if constexpr (std::is_trivially_constructible_v<_OutValueType, _InRefType> && // required operation is trivial
-                  std::is_trivially_default_constructible_v<_OutValueType> &&     // actual operations are trivial
-                  std::is_trivially_assignable_v<_OutRefType, _InRefType>)
+    if constexpr (oneapi::dpl::__internal::__can_avoid_placement_new_in_copy<_OutValueType, _InRefType, _OutRefType>)
```

Comment:
I would change the name to describe the functionality rather than a "purpose". E.g. `__trivial_uninitialized_copy`.

Also, think of the order of arguments; I would use `_OutValueType, _OutRefType, _InRefType` in this case.

Reply:
Done, see 636304f0090feca49c4cedff452e95d77162ee6f.